tests/integration/cartography/intel/aws/test_init.py

```diff
@@ -91,6 +92,29 @@ def test_start_aws_ingestion(mock_run_analysis, mock_sync_multiple, mock_orgs, m
     assert mock_run_analysis.call_count == 3
 
 
+@mock.patch('cartography.intel.aws.boto3.Session')
+@mock.patch('cartography.intel.aws.organizations.get_aws_accounts_from_botocore_config')
+@mock.patch.object(cartography.intel.aws, '_sync_one_account', return_value=None)
+@mock.patch.object(cartography.intel.aws, 'run_analysis_job')
+@mock.patch.object(cartography.intel.aws, 'run_cleanup_job')
+def test_start_aws_ingestion_raises_aggregated_exceptions(
```

Comment:
Two more important things I think we should test for:

- The loop continues after a failure.
- The details like the Account ID and Exception class are logged.

Reply:
I think the loop continuity is covered by this (since all iterations are throwing the exception as a side effect):
https://github.com/lyft/cartography/blob/43b4b95faf02adf728ebd81503364458e86ce5d8/tests/integration/cartography/intel/aws/test_init.py#L113
or do you have an additional scenario in mind?

Also this checks for the Exception class in the traceback, so maybe just add the account ID as discussed above (?):
https://github.com/lyft/cartography/blob/43b4b95faf02adf728ebd81503364458e86ce5d8/tests/integration/cartography/intel/aws/test_init.py#L112

Reply:
@erickduran perfect